README.md

```diff
@@ -106,13 +106,8 @@ acceptable.
 ### API
 
 ```webidl
-enum EffectState {
-   "disabled",
-   "enabled"
-}
-
 dictionary MediaEffectInfo {
-  readonly EffectState state;
+  readonly boolean isEnabled;
 }
 
```

Comment:
```suggestion
partial dictionary BackgroundBlur : MediaEffectInfo { }
```

Reply:
To establish the pattern of per-effect dictionaries, change the type of backgroundBlur to `BackgroundBlur` below.